README.md

```diff
@@ -1740,4 +1740,7 @@ I'm [Alyona](https://github.com/askryp).
 ---
 # Hello everybody!
 I'm [Serge](https://github.com/sergesd). It's really nice to get new and important knowledge in such a nice format and see a lot of progress from everybody on the Telegram channel. It's inspiring!
-![Cat](assets/images/CatFromLviv.jpg)
\ No newline at end of file
+![Cat](assets/images/CatFromLviv.jpg)
```

Comment:
Is there another whitespace deleted?
I turned off this option in my editor, so it shouldn't have done it.